NEWS.md

```diff
@@ -11,6 +11,9 @@
 - All admiral utility functions are exported now such that they can be used in
 admiral extension packages (#1079)
 
+- Moved all developer-facing functions to `{admiraldev}`. This supersedes above (#1079)
+ update (#1231)
```

Comment:
We should remove #1079 from the NEWS file and change the wording here accordingly.